### PR TITLE
[plugin.video.intergalacticfm] 2.1.1

### PR DESCRIPTION
**Pull request by dromer**
### Description
Version 2.1.1
Mtv url fix and some images improved

Version 2.1.0
Some channels added and some removed

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.intergalacticfm/README.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ Stepts to take to install from the Kodi main repository are:
 
 Instructions for installing Kodi can be found here: https://kodi.tv/download
 
+#### Android
+Please consider installing [Kodi](https://f-droid.org/en/packages/org.xbmc.kodi/)
```
Comment (Rechi) [inline plugin.video.intergalacticfm/README.md]: I wouldn't recommend installing Kodi from F-Droid. It doesn't provide the most recent stable release.

Comment (PanderMusubi) [reply plugin.video.intergalacticfm/README.md]: Normally, F-Droid automatically builds when there is a new tagged release availabe.

Comment (Rechi) [reply plugin.video.intergalacticfm/README.md]: see https://github.com/xbmc/xbmc/releases, https://play.google.com/store/apps/details?id=org.xbmc.kodi and https://f-droid.org/en/packages/org.xbmc.kodi/

Comment (dromer) [reply plugin.video.intergalacticfm/README.md]: Hmmm, seems they have AutoUpdateMode turned off indeed: https://gitlab.com/fdroid/fdroiddata/blob/master/metadata/org.xbmc.kodi.yml#L133

If you think we should remove any links for android users we can omit it from the readme.

Comment (PanderMusubi) [reply plugin.video.intergalacticfm/README.md]: Fine by me if that needs to be removed from README.md (also the audio plugin then).

Comment (Rechi) [reply plugin.video.intergalacticfm/README.md]: Yes, in instructions for how to get/intstall Kodi please only link to sites and stores controlled by Team Kodi.

**Pull request closed without merging**